client/ayon_core/hosts/max/plugins/publish/validate_extended_viewport.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from ayon_core.pipeline import PublishValidationError
+from pymxs import runtime as rt
+
+
+class ValidateExtendedViewport(pyblish.api.InstancePlugin):
+    """Validate if the first viewport is an extended viewport."""
+
+    order = pyblish.api.ValidatorOrder
+    families = ["review"]
+    hosts = ["max"]
+    label = "Validate Extended Viewport"
+
+    def process(self, instance):
+        try:
+            rt.viewport.activeViewportEx(1)
+        except RuntimeError:
+            raise PublishValidationError(
+                "Please make sure one viewport is not an extended viewport", title=self.label)
```

Comment:
Preferably this provides the `description` argument explaining what is wrong with it being an extended viewport (like WHY must it not be an extended viewport)

Even better if this could explain how the user can make it a **not** extended viewport. I don't know 3DsMax so do not know what an extended viewport is.

Also, is there any way to disable extended viewport - and re-enable it. If so, what's holding us from just disabling it in a context manager so the review still works fine? (Again, might be stupid question; I don't use max)

Reply:
Understand. Thank you for your feedback! I'll update the PublishValidationError to include the way people could resolve it in 3dsMax.

So an extended viewport are special viewports that are not all supported by 3dsmax. This could be for example if you use redshift, you could have redshift renderview docked as one of the viewports. 
![image](https://github.com/ynput/ayon-core/assets/73248638/957f39c3-2e7b-4c44-9bff-5b8a57b42a05)
![image](https://github.com/ynput/ayon-core/assets/73248638/c1bbc83b-bb1a-484b-99aa-ca39cc2ea08c)

The above example is alright and the validator won't see that as a problem as there is still one available viewport that is a 3dsmax viewport that programmatically we have access to it.

So the problem comes about if all available viewports are extended viewports. Like the image below.
![image](https://github.com/ynput/ayon-core/assets/73248638/ab4d3e08-a70c-42f0-af9e-6e7baedf0ab9)
Cause there isn't any functions exposed to maxscript or python that allows setting or unsetting the viewport. You'll have to do it manually by using the UI.

![image](https://github.com/ynput/ayon-core/assets/73248638/8867874c-6e35-4e19-b2d0-126a604ea96a)
![image](https://github.com/ynput/ayon-core/assets/73248638/12dbc9cc-21c0-45b9-ab84-f45c263c7ce2)